meetings/2023-01-04.md

```diff
@@ -29,36 +39,63 @@
+
 
 ### Add TR/2022/protocol-20221231
 https://github.com/solid/specification/pull/492
 
+* SC: Proposal to add the release version 0.10 of the Solid Protocol. Includes the change log since release 0.9 (approximately 1 year ago). Same repository context as previous TR discussed. TRs should be against the editors draft (https://solidproject.org/ED/protocol), not the specification document. 
 
-### Index and Registry
-* SC: In order to document collections of values or other data that TRs use, a separate document can be published. For example, https://solidproject.org/TR/notifications-protocol refers to https://solidproject.org/TR/notification-subscription-types (Living Document) where an index of subscription types can be found.
-* SC: To date, the index document only listed documents published under `TR/`.
-* SC: The notifications-panel also has https://github.com/solid/notifications/blob/main/README.md#subscription-types that lists all types.
-* SC: Question. Would it be simpler to re-use https://solidproject.org/TR/ as the index for auxiliary documents, e.g., notification channel types, since it is intended to list all of the CG's Work Items. The "Indexing Procedure" can be part of TRs that need it.
-
-
----
 
-### Propose Topic Title
-URL:
+### Publishing Rules for Community Group Documents
 
-* Proposed by: name
-* name: text
+* SC: We have not released a community group draft as recommend by W3C. There is a recommend W3C template, but it does not have community group branding. In the past this was a group decision to release the specs are they are as opposed to a community group draft. The question is: should we or do we need to do that for any spec that will be worked by the community group (e.g., Solid specification).
+* WT: Is it just template/style issue, or a structure issue?
+* SC: Mostly templating rules. Publishing, status, copyrights, patent policy, etc., differ. As far as content specific to the protocol/topic, that does not change.
+* PC: Should we do that? I think no — it's not required. If the group is okay with the current approach, then it is okay. The most important aspect is copyrights, etc. The protocol specification already points to the content contributor rules. The Final Report needs to follow the rules of W3C, etc. It has the added benefit of published on the public W3C website. 
+* SC: we have solidproject.org/TR/. If and when the protocol is published it will be under w3c.org/TR/.
```

Comment:
```suggestion
* SC: we have [solidproject.org/TR/](https://solidproject.org/TR). If and when the protocol is published it will be under [w3c.org/TR/](https://www.w3.org/TR/).
```